search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -27,16 +27,22 @@ function onClickRegion(e) {
   if (e.feature.getProperty('country_data') == null) {
     return;
   }
-  $('#region-info-modal').modal('show');
 
   // Update Modal with information for relevant country.
   const countryName = e.feature.getProperty('name');
   const countryId = e.feature.getId();
+  openModal(countryId, countryName);
+}
+
+function openModal(countryCode, countryName) {
```

Comment:
One suggestion: Can we center the map to that country when opening the modal?

Reply:
I like the idea. My only concern is that we right now don't have coordinates for a country to center it with. So I'm not sure what I would pass as the latlng to center the map.

Reply:
There's definitely data on this online, but yeah only if you have extra time

Reply:
You think that this feature is worth adding a whole other json file?

Reply:
If we move forward with this, it can be in a separate PR.